topics/variant-analysis/tutorials/MTB_transmission/tutorial.md

```diff
@@ -0,0 +1,812 @@
+---
+layout: tutorial_hands_on
+
+title: 'Identifying tuberculosis transmission links: from SNPs to transmission clusters'
+zenodo_link: ''
+objectives:
+- Create a SNP alignment
+- Calculate pairwise SNP distances between MTB samples
+- Identify transmission clusters based on SNP distances
+- Study the emergence and spread of drug resistance based on transmission analysis.
+time_estimation: 2H
+enable: true
+level : Intermediate
+key_points:
+- Clustering is a useful tool to detect transmission links between patients and oubreak investigation.
+- Clustering can be used to investigate the transmission of certain traits, like drug resistance.
+- Clustering does not provide information about particular transmission events nor their directionality (who infected whom).
+- Clustering is very much influenced by sampling. Lower sampling proportions and shorter sampling timeframes lead to lower clustering rates that shoud not be confounded with lack of transmission.
+contributors:
+- GaloGS
+- dbrites
+- cstritt
+---
+
+# Pre-requisites
+This tutorial belongs to *Day 3: Evolutionary epidemiology: using phylogenetics to understand DR emergence and Mtb transmission* of the workshop [Mycobacterium tuberculosis NGS made easy: data analysis step-by-step](https://gallantries.github.io/video-library/events/mtb-ngs/program.html). It assumes you went through the materials
+of *Day 2*, and you have watched the respective webinars and completed the tutorial on [MTB variant analysis](https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html).
```

Comment:
We generally avoid mentioning events things are part of, these could be better expressed through pre-requisites (see other tutorials)